### ISSUE TITLE
How to show hidden elements in monitor mode?

### ISSUE THREAD

**Issue by yinrong** [XuehaiPan/nvitop#9]
no instruction found in README or 'h'

**Comment by XuehaiPan** [comment]
**tl;dr**: run `nvitop` with `nvitop -m full` or simply press <kbd>f</kbd> key. You can set environment variable `NVITOP_MONITOR_MODE="full"` in your shell profile, then when you run `nvitop -m` (without arguments) will go into the full monitor mode.

**NOTE:** the bar plot requires the terminal width is no less than 100.

------

According to the size of the terminal window, some information is hidden when the window is overfull. 

**Render strategy for terminal height:**

Monitor modes (shorcut <kbd>a</kbd> / <kbd>f</kbd> / <kbd>c</kbd>):

1. **`auto` (default):** automaticly resize panels according to the terminal window size. This is the default behavior of `nvitop -m` if you don't pass an explicit argument and haven't set the `NVITOP_MONITOR_MODE` environment variable.

    Resizing priority: process panel > host panel > device panel. The bar plot of GPU utilization is shown in the device panel.

    The `auto` mode is identical to `full` when the terminal size is big enough.

2. **`full`:** show everything but sometimes overfull the terminal window (use keys <kbd>[</kbd> / <kbd>]</kbd> to scroll up and down the entire screen).

3. **`compact`:** only show key information, useful for small windows.

**Render strategy for terminal width:**

`nvitop` requires the terminal width is no less than 79. For the elements on the right side, (bar plots and graph plots for GPU / VRAM percent) they require terminal width >= 100 (otherwise hidden).

**Comment by XuehaiPan** [comment]
Here is a screencast for auto-resizing:

![screencast](https://user-images.githubusercontent.com/16078332/142595126-6f0c0788-6287-44c0-9eaf-a846c6cc10be.gif)

**Comment by XuehaiPan** [comment]
Thanks for the feedback from @yinrong! Feel free to reopen this issue if you have any other questions.

**Comment by yinrong** [comment]
not working in linux.
![image](https://user-images.githubusercontent.com/4221569/159861472-7e46557d-8120-4ab4-ac08-3b3891165ff7.png)
